polygeist/tools/cgeist/Lib/CGExpr.cc

```diff
@@ -2793,7 +2803,7 @@ ValueCategory MLIRScanner::EmitBinDiv(const BinOpInfo &Info) {
             << "Not applying OpenCL/HIP precision options.\n";
       }
     });
-    return LHS.FDiv(Builder, Loc, RHS);
+    return LHS.FDiv(Builder, Loc, RHS, getFastMathFlags(Info.getFPFeatures()));
```

Comment:
Nit: The expression `getFastMathFlags(Info.getFPFeatures())` appears quite often, would it make sense to introduce a helper for it?